modules/account-lib/test/unit/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -136,6 +137,23 @@ describe('TSS ECDSA TESTS', function () {
         }
       }
     });
+
+    it('should derive unhardened child keys', async function () {
+      // const aKeyCombine = MPC.keyCombine(A.pShare, [B.nShares[1], C.nShares[1]]);
+      const aKeyCombine = keyShares[0];
+      const commonKeychain = aKeyCombine.xShare.y + aKeyCombine.xShare.chaincode;
+
+      for (let index = 0; index < 10; index++) {
+        const path = `m/0/1/${index}`;
+        const derive1: string = MPC.deriveUnhardened(commonKeychain, path);
+        const subkey = MPC.keyDerive(A.pShare, [B.nShares[1], C.nShares[1]], path);
+        const derive2 = MPC.deriveUnhardened(commonKeychain, path);
+        // const derivedPk = derive1.slice(0, 64);
+
+        derive1.should.equal(derive2, 'derivation should be deterministic');
+        (subkey.xShare.y + subkey.xShare.chaincode).should.equal(derive1);
+      }
+    });
   });
```

Comment:
not sure why this test is failing on.. any ideas? 🤔 
`(subkey.xShare.y + subkey.xShare.chaincode).should.equal(derive1);`